.envrc

```diff
@@ -0,0 +1 @@
+export RUST_LOG=voxel_rs_client=debug,voxel_rs_common=debug,voxel_rs_server=debug,wgpu_core=warn,gfx_backend_vulkan=warn
```

Comment:
I'm not fond of having this file in the tree, I think it should be removed. You can add it to the .gitignore though! When this is merged, I will add a note in the README.